spec/features/account_history_spec.rb

```diff
@@ -1,24 +1,39 @@
 require 'rails_helper'
 
+RSpec::Matchers.define :appear_before do |later_content|
```

Comment:
Is this matcher ok hanging out in this spec, or do we want to move it into `spec/support/matchers`, or something else?

Reply:
moved into a separate support matcher